.github/workflows/ci.yml

```diff
@@ -6,50 +6,80 @@ on:
 
 env:
   BUILD_TYPE: Release
+  WORKING_DIRECTORY: ${{ github.workspace }}/build
 
 jobs:
   build-mingw:
     runs-on: ubuntu-24.04
 
     steps:
-
     - name: Install packages
       run: |
         sudo dpkg --add-architecture i386
         sudo apt-get update
         sudo apt-get install --no-install-recommends g++-mingw-w64-i686-posix ninja-build wine32 wine
+      shell: bash
 
     - uses: actions/checkout@v4
 
+    - name: Set Current Git Commit Head SHA
+      run: echo "GIT_CURRENT_COMMIT_HEAD_SHA=$(git rev-parse --short=10 HEAD)" >> $GITHUB_ENV
```

Comment:
`--short` by default uses 8 characters, that's probably enough?

Reply:
Agreed. In fact, I will remove this flag as it is optional.